members/CusA7529f3f51544548978.yaml

```diff
@@ -3,43 +3,50 @@ contact_form:
   method: post
   action: ""
   steps:
-    - visit: "https://www.regulations.gov/comment?D=CMS-2018-0135-0001"
+    - visit: "https://www.federalregister.gov/documents/2018/11/09/2018-24504/patient-protection-and-affordable-care-act-exchange-program-integrity#open-comment"
     - wait:
         - value: 5
-    - click_on:
-        - selector: input#gwt-uid-164
-    - wait:
-        - value: 1
     - find:
-        - selector: div.GIY1LSJGSC form
+        - selector: select#comment_country
+    - select:
+        - name: Country
+          selector: select#comment_country
+          value: United States
+          required: true
```

Comment:
`options`?

Reply:
do we need an `options` step for this `select`, or just hard-coding United States would suffice? I tested with the code as above and it works fine

Reply:
We don't actually need an options, no.